ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -460,6 +460,25 @@ spec:
         - name: GITHUB_TOKEN_SECRET_KEY
           value: "$(params.github_token_secret_key)"
 
+    # Add the connect page URL in github PR comment to view the test results/logs
+    - name: github-add-connect-url-comment
```

Comment:
Wouldn't this lead to two identical comments posted, after both hosted and release pipelines are run?

Reply:
It seems so. It'd be nicer to have distinct comments. For example, a user shouldn't care to see test results after the release pipeline succeeds or fails. If this pipeline is running they've already moved past certification but they may still wish to look at the release pipeline logs.

Reply:
As per my understanding, we need to add the two comments for the same link anyway, just need to distinguish the comment message. If this is what we wanted, then what if we just add the prefix `hosted or release` pipeline in the comment message?

Reply:
Or maybe we drop this comment from the release pipeline altogether. It doesn't seem to add any value and the message at the moment is generic enough for all use cases. Wdyt?

Reply:
I think it would be better to just drop the message from the Release pipeline.

Reply:
Agreed. No point posting the same link again.

Reply:
Okay, then I will remove the comment from the release pipeline.